client/landing/stepper/declarative-flow/internals/components/step-route/index.tsx

```diff
@@ -16,6 +17,14 @@ type StepRouteProps = {
 	renderStep: ( step: StepperStep ) => JSX.Element | null;
 };
 
+const PerformanceTrackingStop = () => {
```

Comment:
We can't use the already existent `PerformanceTrackerStop` because it relies on the calypso store to capture the current "page" and we don't have this concept on Stepper.

We will need to implement other versions of the same components and hooks without relay on calypso store.

The good part they probably will be compatible with other clients that are using react-router.